www/doc/1.6/api-endpoints.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
```

Comment:
Great idea to add documentation, but you only need to add it to 1.7. Older releases won't have the API!

Reply:
Noted, I'll remove the 1.6 copy.